### DISCUSSION TITLE
Build and ship strategy

### DISCUSSION THREAD

**Discussion by klimashkin** [uNetworking/uWebSockets.js#274] [Q&A] [answered]
Hi @alexhultman!

That is probably an easy question, but I'm a little bit confused.
uWebSockets.js comes with the prebuilt binaries for a various of platforms. And in our case we build deliverable bundle inside a docker image in Jenkins and then ship the result as an rpm package to a given system, usually CentOS/ RHEL 6-7.X.
How can I be sure that that specific OS is supported? Is there a mapping table for linux distributives, or it's maybe based on kernel version or glibc version?

**Comment by ghost** [comment, marked as answer]
Binary compatibility is most likely limited by glibc, but you can always build your own binaries on older Linux systems to get wider compatibility.
